mne/tests/test_epochs.py

```diff
@@ -101,23 +101,22 @@ def test_average_movements():
     # usable data
     crop = 0., 10.
     origin = (0., 0., 0.04)
-    raw = read_raw_fif(fname_raw_move, allow_maxshield='yes',
-                       add_eeg_ref=False)
+    raw = read_raw_fif(fname_raw_move, allow_maxshield='yes')
     raw.info['bads'] += ['MEG2443']  # mark some bad MEG channel
-    raw.crop(*crop, copy=False).load_data()
-    raw.filter(None, 20, method='iir')
+    raw.crop(*crop).load_data()
+    raw.filter(None, 20)
```

Comment:
No need to filter with IIR, and the test with IIR mode doesn't actually work on old versions of SciPy. Easier to make it use FIR than to add a decorator (and decrease coverage on those runs)